### PR TITLE
Checkbashisms

### PR DESCRIPTION
**Pull request by maelle**
start work on #46

### CODE REVIEW

**Comment by maelle** [comment]
A symlink won't work :-( "COPY failed: Forbidden path outside the build context: checkbashisms ()"

**Comment by maelle** [comment]
@gaborcsardi I've now built the Ubuntu image and it seems checkbashisms is there but I haven't _used_ it

```
(base) maelle@maelle:~/Documents/R-hub/rhub-linux-builders$ docker run -it 979eb558e1b4
root@2cbd8a4ab959:/# checkbashisms
Usage: checkbashisms [-n] [-f] [-x] script ...
   or: checkbashisms --help
   or: checkbashisms --version
This script performs basic checks for the presence of bashisms
in /bin/sh scripts and the lack of bashisms in /bin/bash ones.
```

**Comment by maelle** [comment]
I did the same check as above for Fedora (built the image, ran it interactively to see whether checkbashisms was available, but didn't _use_ checkbashisms).

**Comment by maelle** [comment]
The error I get for Debian is `E: Unable to locate package texlive-generic-recommended`

Why is it "texlive-generic-recommended" and not "texlive-plain-generic"?

**Comment by maelle** [comment]
@gaborcsardi the latest build of the Debian R-hub image on Docker was 9 months ago, were you able to build it locally since then? I'm hoping you might get the same error message. :-)

**Comment by maelle** [comment]
FWIW if I replace "texlive-generic-recommended" with "texlive-plain-generic" I can build the image, and in it I can find checkbashisms, but I suppose it might break LaTeX stuff at the same time.

**Comment by gaborcsardi** [comment]
> FWIW if I replace "texlive-generic-recommended" with "texlive-plain-generic" I can build the image, and in it I can find checkbashisms, but I suppose it might break LaTeX stuff at the same time.

Seems like that's the correct package indeed.

File: debian/Dockerfile
Comment (maelle) [inline debian/Dockerfile:58]: made the change @gaborcsardi

**Comment by maelle** [comment]
@gaborcsardi is there anything else I should do?

File: debian/Dockerfile
Comment (gaborcsardi) [inline debian/Dockerfile]: Can you please remove these comments? Thanks.

**Review by gaborcsardi — COMMENTED** [review]
Thanks!

File: debian/Dockerfile
Comment (maelle) [reply debian/Dockerfile]: :heavy_check_mark:

**Review by gaborcsardi — APPROVED** [review]

**Comment by maelle** [comment]
Can I merge this @gaborcsardi? And once it's merged, what does it take to deploy updated images?

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**
